### PR TITLE
DEV-7961: Login Banner

### PR DESCRIPTION
**Pull request by dpb-bah**
**High level description:**

Adding support for a dynamic banner to appear on the login page. 

**Technical details:**

N/A

**Link to JIRA Ticket:**

[DEV-7961](https://federal-spending-transparency.atlassian.net/browse/DEV-7961)

**Mockup**
N/A

The following are ALL required for the PR to be merged:

Author: 
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled Demo including Design/Testing/Front-end OR Provided Instructions for Local Testing above and in JIRA
- [x] Verified cross-browser compatibility
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)

Reviewer(s):
- [x] Design review completed
- [ ] Frontend review completed
- [ ] Merged concurrently with [Backend#2217](https://github.com/fedspendingtransparency/data-act-broker-backend/pull/2217)

### CODE REVIEW

**Review by alburde1 — COMMENTED** [review]
Why are we creating a brand new component and reinventing the wheel instead of just using the existing `Banner` component here?

**Comment by dpb-bah** [comment]
* Updated to reuse Banner component
* Updated Banner component styles to be consistent and supporting of success and error banners.

File: src/_scss/layouts/login/loginForm/loginForm.scss [hunk 1]
```diff
@@ -58,7 +56,19 @@
...
+		@include loginTopBanner;
+
+        padding: rem(15);
+        margin-bottom: rem(20);
+        border-bottom: 0 !important;
+
+        .container {
+            width: 100%;
+        }
+	}
```
Comment (alburde1) [inline src/_scss/layouts/login/loginForm/loginForm.scss:71]: Tabbing issues

File: src/js/containers/login/LoginContainer.jsx [hunk 2]
```diff
@@ -8,11 +8,13 @@ import PropTypes from 'prop-types';
...
 import { connect } from 'react-redux';
 
-import * as LoginHelper from 'helpers/loginHelper';
 import LoginPanel from 'components/login/LoginPanel';
 import LoginMax from 'components/login/LoginMax';
+import Banner from 'components/SharedComponents/Banner';
+import * as LoginHelper from 'helpers/loginHelper';
 import * as sessionActions from 'redux/actions/sessionActions';
 import { kGlobalConstants } from '../../GlobalConstants';
+import * as ReviewHelper from '../../helpers/reviewHelper';
```
Comment (alburde1) [inline src/js/containers/login/LoginContainer.jsx:17]: Did we need to import reviewHelper here? Based on these changes it doesn't seem like we use it.

**Review by alburde1 — CHANGES_REQUESTED** [review]

**Review by alburde1 — APPROVED** [review]

**Pull request merged**